src/build/mod.rs

```diff
@@ -35,11 +35,11 @@ pub fn build(args: &BuildArgs) -> anyhow::Result<()> {
     Ok(())
 }
 
-pub(crate) fn ensure_web_setup() -> anyhow::Result<()> {
+pub(crate) fn ensure_web_setup(silent: bool) -> anyhow::Result<()> {
     // `wasm32-unknown-unknown` compilation target
-    rustup::install_target_if_needed("wasm32-unknown-unknown")?;
+    rustup::install_target_if_needed("wasm32-unknown-unknown", silent)?;
     // `wasm-bindgen-cli` for bundling
-    cargo::install::if_needed(wasm_bindgen::PROGRAM, wasm_bindgen::PACKAGE, true, false)?;
+    cargo::install::if_needed(wasm_bindgen::PROGRAM, wasm_bindgen::PACKAGE, !silent, false)?;
```

Comment:
I would like to invert the boolean behavior here:

from (ask if true):
```rs
fn if_needed(program: &str, package: &str, ask_user: bool, hidden: bool) -> anyhow::Result<bool>
```
to (ask if false)
```rs
fn if_needed(program: &str, package: &str, silent: bool, hidden: bool) -> anyhow::Result<bool>
```